packages/serverless-offline-sqs/src/index.js

```diff
@@ -208,7 +208,10 @@ class ServerlessOfflineSQS {
       await fromCallback(cb => client.createQueue(params, cb));
     }
 
-    const {QueueUrl} = await fromCallback(cb => client.getQueueUrl({QueueName}, cb));
+    let {QueueUrl} = await fromCallback(cb => client.getQueueUrl({QueueName}, cb));
+
+    const endpointWithoutHost = QueueUrl.replace(/^[a-z]{4,5}:\/{2}[a-z]{1,}:\d{1,4}.(.*)/, '$1');
+    QueueUrl = `${this.getConfig().endpoint}/${endpointWithoutHost}`;
```

Comment:
Perhaps we could remove the request to AWS completely?

```js
const QueueUrl = `${this.getConfig().endpoint}/queue/${QueueName}`;
```

Reply:
I was about to reply this to the main pr.

The reason behind this is that it does not have to necessarily be the same endpoint for each message broker. 

AWS SQS does the following:
`<host>/<id>/<queue-name>`

This does not apply to elasticMq, which does:
`<host>/queue/<queue-name`

Therefore the request is necessary, but the hostname will always be the same. So we do the request to fetch the path of the endpoint. And then replace it with the provided hostname/endpoint.

Reply:
Good point. Cheers for the explanation :)